src/app/containers/Footer/index.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Footer from '../../components/Footer';
+
+const externalLink = {
```

Comment:
I think we should concatenate all of this data into a single `data` object with properties, so it forms a model of data more coherently.

Reply:
We'd have to destructure them on the first line of the component, so I don't think it would add much.  If we were to rename the props to be `links`, `copyrightText`, `externalLink` as before, would that be sufficient?

Reply:
You could just destructure `data` into the `Footer` component though: `{...data}`. It is better if the props are more specific, I just think it is a bit tidier to keep the model of data in a single data structure - I don't feel too strongly.